.gitlab-ci.yml

```diff
@@ -53,3 +63,8 @@ test-3.6:
   extends: .linux-test-base
   variables:
     PYTHON_VERSION: "3.6"
+
+test-3.6-win:
+  extends: .windows-test-base
+  tags:
```

Comment:
You could specify the PYTHON_VERSION in the windows test job (as is being done for the Linux test jobs), then in then .windows-test-base verify that the installed version matches the PYTHON_VERSION  value.

Reply:
implemented